api/v2.0/swagger.yaml

```diff
@@ -4174,6 +4174,170 @@ paths:
           $ref: '#/responses/403'
         '500':
           $ref: '#/responses/500'
+  /system/purge:
```

Comment:
Will we expand the purge to support other kinds of resources? If not, we can use /system/purgeAudit?

Reply:
Updated